downstream/modules/platform/ref-controller-automation-analytics.adoc

```diff
@@ -2,7 +2,7 @@
 
 = {Analytics}
 
-When you imported your license for the first time, you were given options related to the collection of data that powers {Analytics}, a cloud service that is part of the {PlatformNameShort} subscription.
+When you imported your license for the first time, you were given the option of collecting data that powers {Analytics}, a cloud service that is part of the {PlatformNameShort} subscription.
```

Comment:
This is changed now. There is no option anymore when logging in or importing the license. See my draft here for how it works now and let me know if you have questions: https://docs.google.com/document/d/1mEYguv3V0WUUHBMc8tsgqlK8RB3ro8IZSlQmMuFfhww/edit#bookmark=id.mpul3xd3av0x

Reply:
Updated.